### DISCUSSION TITLE
Release note for Pika

### DISCUSSION THREAD

**Discussion by databill86** [pika/pika#1381] [General]
Hello, 

Is it possible for you guys to add some documentation for the next releases ? 
It's a little bit complicated to check all the changes that are made from one version to another without a change log or a release note (have to go through this for every release: https://github.com/pika/pika/compare/1.2.1...1.3.0, https://github.com/pika/pika/milestone/19?closed=1 ). 

Thank you very much!

**Comment by lukebakken** [comment]
If there are breaking API changes, they will be documented here:

https://github.com/pika/pika/blob/main/CHANGELOG.rst

Otherwise, the milestone and diff are what we have time to deliver. Thanks for using Pika!
